lib/ModelMixins/TableMixin.ts

```diff
@@ -175,6 +176,29 @@ export default function TableMixin<T extends Constructor<Model<TableTraits>>>(
       });
     }
 
+    /*
+     * Appends new table data in column major format to this table.
+     * It is assumed that thhe column order is the same for both the tables.
+     */
+    append(dataColumnMajor2: string[][]) {
+      if (
+        this.dataColumnMajor !== undefined &&
+        this.dataColumnMajor.length !== dataColumnMajor2.length
+      ) {
+        throw new DeveloperError(
+          "Cannot add tables with different numbers of columns."
+        );
+      }
+
+      runInAction(() => {
```

Comment:
Consider adding the `@action` decorator to this method instead of using `runInAction`.